routes/index.js

```diff
@@ -3,13 +3,95 @@ const router = express.Router();
 const queries = require('../queries');
 
 
-router.get('/', queries.getAllBooks);
-router.get('/author/:author', queries.getSingleAuthor);
-router.get('/genre/:genre', queries.getSingleGenre);
-router.get('/book/:id', queries.getSingleBook);
-router.get('/published/:publish_year', queries.getSinglePublishYear);
-router.get('/post', queries.addingBook);
-router.post('/post', queries.createBook)
+router.get('/', ( request, response, next ) => {
+  queries.getAllBooks()
+  .then( data => {
```

Comment:
I like how you moved handling of the response out of queries and into routes. Good separation of concerns. You could even go a step further and break out the whole 
```
queries.getAllBooks()
 .then( data => {
   response.render('index', {
    title: "Wreka Stow",
    books: data
  })
})
.catch( function(err) {
 return next(err)
})
});
```
into a separate function in a separate folder called something like `controllers` or `routeHandlers` and import them in to your routes folder. All the cools kids have good separation of concerns <3